src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -17,6 +17,19 @@
 {% endblock %}
 
 {% block content %}
+  <div id="app">
```

Comment:
I assume this is leftover test data?

Reply:
No, vue needs an id to define where it should process the components

Reply:
@bjorvack your id is `checkbox-id="1234-check"` ...
This is a random checkbox in the edit page of the blog so can you give me a reason why this isn't test data?